lambdas/src/Server.ts

```diff
@@ -88,3 +100,56 @@ export class Server {
     }
   }
 }
+
+/*
+
+https://peers.decentraland.org/lambdas/wearables/erc721/<contract>/<option>/<emission>
```

Comment:
What is all this? Can we delete it? Some of it is in spanish

Reply:
Ups...